code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -169,3 +175,84 @@
 			output += "[IO.name] - <span style='color:green;'>OK</span>\n"
 
 	src << output
+
+/mob/living/carbon/human
+	var/next_sonar_ping = 0
+
+/mob/living/carbon/human/proc/sonar_ping()
```

Comment:
Who gets this? o: Should mention it too. Might want to even give it to Xenochimera if it's like only Vox or something, since they have echolocation.